guides/source/configuring.md

```diff
@@ -829,6 +829,8 @@ production:
 
 Now the behavior is clear, that we are only using the connection information in `ENV['DATABASE_URL']`.
 
+NOTE: When we are using `ENV['DATABASE_URL']` for connection management, it gets applied to the current Rails environment. This is in contrast with the `database.yml` where we have to explicitly add database configuration for different environments. By default, the `ENV['DATABASE_URL']` gets applied to `development` environment if Rails environment is not set.
```

Comment:
Yeah, it may not be the fault of your explanation, it may just be that's behavior is kind of crazy 😆